src/main/java/com/alkemy/ong/controller/UserController.java

```diff
@@ -19,4 +19,15 @@
         this.userService.deleteUserById(id);
         return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
     }
+    @PatchMapping("/user/{id}")
```

Comment:
Did you add the PatchMapping to the rout in the security config?

Reply:
not add pathMapping, change now and oder modified